sse2neon.h

```diff
@@ -3439,6 +3439,27 @@ FORCE_INLINE int _mm_comieq_sd(__m128d a, __m128d b)
 #endif
 }
 
+// Compare the lower double-precision (64-bit) floating-point element in a and b
+// for not-equal, and return the boolean result (0 or 1).
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_comineq_sd
+FORCE_INLINE int _mm_comineq_sd(__m128d a, __m128d b)
+{
+#if defined(__aarch64__)
+    return !vgetq_lane_u64(vceqq_f64(a, b), 0);
+#else
+    uint32x4_t a_not_nan =
+        vceqq_u32(vreinterpretq_u32_m128d(a), vreinterpretq_u32_m128d(a));
+    uint32x4_t b_not_nan =
+        vceqq_u32(vreinterpretq_u32_m128d(b), vreinterpretq_u32_m128d(b));
```

Comment:
As you interpret to `u32`, the **NaN** comparison would be meaningless.

Reply:
Should I remove the NaN detection here?
We may leave a `FIXME` here, since I think there is no way for detect double precision NaN.
`_mm_cmpeq_pd` has the same problem too

Reply:
Please leave a `FIXME` here.